### PR TITLE
Pip removal

### PR DESCRIPTION
**Pull request by jburel**
Remove the "pypi" option

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,16 +3,12 @@ OMERO Python Dependencies
 
 Python dependencies for OMERO.
 
-OMERO depends on several Python modules, some of which can be installed from distribution packages or from PyPI (`pip`).
+OMERO depends on several Python modules, they should be installted from distribution packages.
```
Comment (manics) [inline README.md]: typo: `installted`

File: tasks/main.yml [hunk 2]
```diff
@@ -20,12 +14,7 @@
     - python-numexpr
     - python-tables
     - python-pip
-    - PyYAML
+    - python-yaml
```
Comment (manics) [inline tasks/main.yml]: The real name of the RPM package is `PyYAML`, `python-yaml` is effectively an alias.

File: tasks/main.yml [hunk 3]
```diff
@@ -20,12 +14,7 @@
...
     - scipy
+    - python-pillow
```
Comment (manics) [inline tasks/main.yml]: alphabetical order 😄

**Review by manics — CHANGES_REQUESTED** [review]
How does everyone feel about just making all these Python deps compulsory instead of optional?

File: tasks/main.yml (hunk 3 above)
Comment (jburel) [reply tasks/main.yml]: :-)

**Comment by jburel** [comment]
ok for me to have them compulsory since you won't be able to do much if anything w/o them

File: tasks/main.yml (hunk 2 above)
Comment (jburel) [reply tasks/main.yml]: read better with alias, happy to roll back

File: defaults/main.yml [hunk 4]
```diff
@@ -1,7 +1,2 @@
 ---
 # defaults file for roles/omero-python-deps
-
-# Distribution or PyPI packages?
-omero_python_deps_profile: distribution
-# Install minimal or all recommended packages?
-omero_python_deps_recommended: True
```
Comment (manics) [inline defaults/main.yml:7]: You might as well `git rm defaults/main.yml` since there's nothing in it.

**Review by manics — CHANGES_REQUESTED** [review]
Apart from the minor comment this looks good, I've tested this by modifying https://github.com/openmicroscopy/ansible-role-omero-server/blob/master/tests/requirements.yml and running `molecule`

**Comment by jburel** [comment]
@manics done

**Comment by manics** [comment]
I've retested as before, good to merge.

**Comment by joshmoore** [comment]
Proposed version for release?

**Comment by manics** [comment]
How about 1.1 since pypi was effectively a development feature.

**Comment by joshmoore** [comment]
😼

**Pull request merged**